product_docs/docs/postgres_for_kubernetes/1/rel_notes/1_19_0_rel_notes.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "EDB Postgres for Kubernetes 1.19.0 release notes"
+navTitle: "Version 1.19.0"
+---
+
+This release of EDB Postgres for Kubernetes includes the following:
+
+|      Type      |                                                                                  Description                                                                                   |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Upstream merge | Merged with community CloudNativePG 1.19.0. See the community [Release Notes](https://cloudnative-pg.io/documentation/1.19/release_notes/v1.19/).                                    |
+| Feature | New PostgreSQL 15 image containing [PostgreSQL 15.1](https://www.postgresql.org/docs/15/release-15.html).  |
+| Feature  | Support for [Transparent Data Encryption (TDE)](/tde/latest). TDE encrypts, transparently to the user, any user data stored in the database system. |
```

Comment:
"Support for Transparent Data Encryption (TDE) when using EDB Postgres Advanced (EPAS) 15". Or something like that.